### DISCUSSION TITLE
Audio break when you change output device

### DISCUSSION THREAD

**Discussion by AlexHgC** [ppy/osu#15497] [Q&A] [answered]
There is two issues with audio :

- There is 2 default audio

- If i chose my headphone and revert back to default, sound disappear. If i change song, i have the song sound, but UI sound don't work, i have to restart the game to get it back. 

**Here is a video :**

https://user-images.githubusercontent.com/31996023/140606436-a6bcaf53-5d42-47e5-87f0-c35148cea3ef.mp4

**Comment by frenzibyte** [comment]
The issue about audio breaking when switching between output devices sounds similar to https://github.com/ppy/osu/issues/14729, if not a duplicate.

As for the duplicate default audio device, I've created an issue for that #15498 (not sure if duplicate, can't recall it like that)

**Reply by AlexHgC** [reply]
Sound similar but not the totally the same, because changing song will make the other song ( not UI sound ) works for me, and appears after changing output and revert back to default. But maybe the same thing cause the two issues, just different scenarios like peppy said on #14729

**Comment by frenzibyte** [comment, marked as answer]
The remaining "audio breaks when changing output" has now been fixed in https://github.com/ppy/osu-framework/pull/4921 with the usage of the new `BASS_DEVICE_REINIT` flag.
